src/commands/repos/UpdateRepos.ts

```diff
@@ -66,11 +67,14 @@ export class UpdateRepos extends AbstractReposCommand {
             return Promise.reject('No branch or tag given in parent-repos.json for repo: ' + repoName);
         }
 
+        const latestTagOfRelease = await Repository.getLatestTagOfReleaseBranch(repoName, repoProperties);
```

Comment:
If I didnt mess something it could probably be a oneliner:

`repoProperties.latestTagForRelease = await Repository.getLatestTagOfReleaseBranch(repoName, repoProperties);`